<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbRozpocznij">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Rozpocznij</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lw1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>256</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbZakoncz">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Zakończ</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbWyswietlacz">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStatystyki">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Śledź statystki</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pteLista">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUsun">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Usuń Typ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDodaj">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Dodaj Typ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTyp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Typy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Licznik:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wszystkie złote godziny:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbZadaj">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Zadaj liczbę godzin</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbZadaj">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edytuj ostatnią złotą godzinę:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEdytujTyp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Typ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDataRozp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Godzina rozpoczęcia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>591</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../perfumy na lato_jzzpl.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Wakacje coraz bliżej!!!</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbgodz">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbmin">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Godz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDataZakon">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Godzina zakończenia</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbmin2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbgodz2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>pbRozpocznij</zorder>
   <zorder>lw1</zorder>
   <zorder>pbZakoncz</zorder>
   <zorder>tbWyswietlacz</zorder>
   <zorder>pbStatystyki</zorder>
   <zorder>pteLista</zorder>
   <zorder>pbUsun</zorder>
   <zorder>pbDodaj</zorder>
   <zorder>leTyp</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>pbZadaj</zorder>
   <zorder>sbZadaj</zorder>
   <zorder>label_4</zorder>
   <zorder>pbEdytujTyp</zorder>
   <zorder>pbDataRozp</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>sbgodz</zorder>
   <zorder>sbmin</zorder>
   <zorder>label_5</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>pbDataZakon</zorder>
   <zorder>sbmin2</zorder>
   <zorder>sbgodz2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
